dylint_lints/validate_gts_docs.py

```
@@ -0,0 +1,473 @@
+    for ctx in WILDCARD_ALLOWED_CONTEXTS:
+        if ctx in before:
+            return True
+    
+    # Also check for code fence contexts that indicate filter examples
+    if '$filter' in line.lower():
+        return True
+    
+    return False
+
+
+def is_bad_example_context(line: str, match_start: int, prev_lines: List[str] = None) -> bool:
```

Ruff (0.14.11)

Unused function argument: `match_start`

(ARG001)